lib/modules/platform/types.ts

```diff
@@ -83,6 +83,7 @@ export interface Issue {
 export type PlatformPrOptions = {
   azureAutoApprove?: boolean;
   azureWorkItemId?: number;
+  azureAutoWi?: boolean;
```

Comment:
The other option is currently `azureWorkItemId` so I'd prefer to keep it with the full name.

```suggestion
  azureAutoWorkItem?: boolean;
```